tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Resource    ../../../Resources/ODS.robot
+Resource    ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Resource    ../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+
+
+*** Test Cases ***
+Any OpenShift User Should Also Be A RHODS User
+    [Documentation]    Any OpenShift user should also be a RHODS user
+    [Tags]    Sanity
+    ...       ODS-1164
+    Set Library Search Order    SeleniumLibrary    InstalledOperators
+    Open Browser    ${OCP_CONSOLE_URL}    ${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Login To Openshift    ${OCP_ADMIN_USER.USERNAME}    ${OCP_ADMIN_USER.PASSWORD}    ${OCP_ADMIN_USER.AUTH_TYPE}
+    Navigate To Page    Workloads    ConfigMaps
+    Select Rhods-Groups-Config
+    Verify Groups ConfigMap Contains The Expected OCP Groups
+    Spawn A Minimal Notebook
+
+
+*** Keywords ***
+Select Rhods-Groups-Config
+    [Documentation]    Select rhods-groups-config and click
+    Search Last Item Instance By Title In OpenShift Table    rhods-groups-config    redhat-ods-applications
+    Click Link    xpath=//a[contains(text(),'rhods-groups-config')]
+
+Verify Groups ConfigMap Contains The Expected OCP Groups
+    [Documentation]    Verify that ConfigMap contains expected Groups
+    Wait Until Page Contains Element    xpath=//*[@class='co-pre-wrap co-copy-to-clipboard__text']
+    ${version_check} =    Is RHODS Version Greater Or Equal Than    1.8.0
+    IF    ${version_check} == True
+        Element Text Should Be    xpath=//*[@id="content-scrollable"]//dd[1]/div/pre    cluster-admin
```

Comment:
ODS-1164 on polarion reports "dedicated-admins" instead of "cluster-admin". Which one is the correct one?

Reply:
cluster-admin is the last message  in Jira.
https://issues.redhat.com/browse/RHODS-2442